src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/ReadOnlyTensorSpan_1.cs

```diff
@@ -348,7 +348,8 @@ ref Unsafe.Add(ref _reference, linearOffset),
         /// <inheritdoc cref="IReadOnlyTensor{TSelf, T}.TryCopyTo(in TensorSpan{T})" />
         public bool TryCopyTo(scoped in TensorSpan<T> destination)
         {
-            if (TensorShape.AreCompatible(_shape, destination._shape))
+            // TODO: TANNER, THIS WILL NOT WORK.
+            if (TensorShape.AreCompatible(destination._shape, _shape, false))
```

Comment:
I think this TODO is no longer relevant with the change you made.

As a nit, it'd be great if we could use a named parameter for the `false` to improve clarity:
```suggestion
            if (TensorShape.AreCompatible(destination._shape, _shape, allowBidirectional: false))
```